src/postamble.js

```diff
@@ -316,6 +316,10 @@ Module['exit'] = Module.exit = exit;
 var abortDecorators = [];
 
 function abort(what) {
+  if (Module['onRuntimeInitializeFailed']) {
+    setTimeout(function () { Module['onRuntimeInitializeFailed'](what); }, 0);
```

Comment:
why is there a `setTimeout` here? it needs an explanation, but also I don't think it's needed, unless I'm missing something?